packages/replay_cubit/example/ios/Podfile

```diff
@@ -0,0 +1,87 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
I think if we remove `path_provider` we can remove this as well

Reply:
I removed! And do we remove the podfile too?

Reply:
Yeah I think so since there are no pods required 😄

Reply:
Ok sweet! Removed